src/components/utils/include/utils/timer.h

```diff
@@ -66,7 +67,7 @@ class Timer {
    * @param name for indentify of current timer
    * @param task_for_tracking is SPtr to trackable task
    */
-  Timer(const std::string& name, const TimerTask* task_for_tracking);
+  Timer(const std::string& name, TimerTask* task_for_tracking);
```

Comment:
@AGritsevich, we are owner of the object, but not the pointer. So, modification of the pointer looks redundant